.github/workflows/release.yml

```diff
@@ -1,33 +1,55 @@
 name: Release
 
 on:
+  workflow_dispatch:
+    inputs:
+      package:
+        type: choice
+        required: true
+        description: The package to publish.
+        options:
+          - iota-sdk
+          - iota-sdk-crypto
+          - iota-sdk-graphql-client
+          - iota-sdk-graphql-client-build
+          - iota-sdk-transaction-builder
+          - iota-sdk-types
+      version:
+        type: string
+        description: The exact version.
   pull_request:
     types: closed
-    branches:
-      - "release/*/*.*.*-alpha.*"
-      - "release/*/*.*.*-beta.*"
-      - "release/*/*.*.*-rc.*"
-      - "release/*/*.*.*"
+    branches: [develop]
```

Comment:
I feel like this should be possible from GH actions... Maybe another trigger than `pull_request`?